README.md

```diff
@@ -47,54 +30,58 @@ This project is part of Turing Software and Design's Front-End Program, Module 2
+#### RECIPE SEARCH FEATURE
+<img src="src/images/RTWCAppRecipeSearch.png" alt="app searching recipes with 'apple'" width="600">
 
+#### MY RECIPES VIEW
+<img src="src/images/RTWCAppMyRecipesView.png" alt="app my recipes view" width="600">
+
+#### PANTRY FEATURE
+<img src="src/images/RTWCAppPantryModPopup.png" alt="app with pantry and modify pantry menus open" width="600">
 
-<!-- *************** INFO NEEDED ***************** -->
 ## Roadmap
 
 The total time allotted for this project was approximately 12 days. Future features contemplated by the developers include the following:
 
-* Here's something we'd like to see added
-* Here's something we'd like to see added
-* Here's something we'd like to see added
+* Tabbing through the app is functional, but we would love to see ways to improve on the accessibility.
+* We would ideally like to add a feature to add items to a grocery list according to what is needed for a specific recipe. These methods are available on the Recipe class, so this would be a nice next step.
+* Allowing the user to rate a recipe or offer tips/notes after confirming that they have cooked it would be a great addition to the app.
+* Adding an allergen search for recipes would be great for accessibility on a whole new level.
 
 See the [open issues](https://github.com/ericberglund117/Refactor-tractor-WC-EB-KM-CD/issues) for a list of proposed features (and known issues).
 
-
-<!-- CONTACT -->
 ## Contact
 
-Eric Berglund - [Eric's Github Link](https://github.com/ericberglund117)
-
-Kristi Miller - [Kristi's Github Link](https://github.com/Kristiannmiller)
+ [Eric Berglund](https://github.com/ericberglund117)
 
-Chadrick Dickerson - [Chadrick's Github Link](https://github.com/chadrick-d-dev)
+ [Kristi Miller](https://github.com/Kristiannmiller)
 
-Project Link: [https://github.com/ericberglund117/Refactor-tractor-WC-EB-KM-CD](https://github.com/ericberglund117/Refactor-tractor-WC-EB-KM-CD)
+ [Chadrick Dickerson](https://github.com/chadrick-d-dev)
 
+Project Link: [Refactor-Tractor (What's Cookin') Repo](https://github.com/ericberglund117/Refactor-tractor-WC-EB-KM-CD)
 
 
-<!-- ****************** INFO NEEDED **************** -->
 ## Acknowledgements
 
 Robert Gu - This developer team would like to thank Turing Module 2 anchor instructor Robert Gu for his assistance throughout this project and for his daily lessons on the magic of Javascript.
 
 Travis Rollins - We would also like to thank Turing Module 2 instructor Travis Rollins for his guidance and assistance on this project and the module in general
 
-Our Turing Mentors - Including Turing-wide mentor John Adams, and Turing graduates Jon Sweet, Pol Sieira, and Aimee Fritz for their constant support throughout this challenging project.
+Our Turing Mentors - Including Turing-wide mentor John Adams, and Turing graduates Jon Sweet, Pol Sieira, Eric Fitzsimons, and Aimee Fritz for their constant support throughout this challenging project.
```

Comment:
This might be the most thorough, yet succinct README I've ever been a part of. Thank you for doing this

Reply:
I concur. Very thorough and succinct! Way to go Team!